### PR TITLE
check if cycles are up to date, if not show alert

### PR DESCRIPTION
**Pull request by MichalD96**
fancy feature ;)
function checks if is time to change cycles
if last update date not match today date, it shows alert banner on top

### CODE REVIEW

File: assets/js/cycles.js [hunk 1]
```diff
@@ -23,5 +23,25 @@ var Cycles = {
...
+}
+
+// show alert when cycles are not up to date
+setInterval(function () {
+    var newDate = new Date(),
+        day = newDate.getUTCDate(),
+        hour = newDate.getUTCHours(),
+        minute = newDate.getUTCMinutes();
+    
+    if (hour < 1 && minute <= 20)
```
Comment (Senexis) [inline assets/js/cycles.js]: Why this check? As long as it hasn't been updated "today" (in UTC) it's fine, no need to force us to update it in 20 minutes. ;-)

File: assets/js/cycles.js [hunk 2]
```diff
@@ -23,5 +23,25 @@ var Cycles = {
...
+        $('.map-alert').css('opacity', '1');
+    
+    if (day != Cycles.setLocaleDate()) {
+        $('.map-alert').css('opacity', '1');
+        return;
```
Comment (Senexis) [inline assets/js/cycles.js]: Not needed as there's nothing after it to do. Allows you to remove the `{` and `}` as well.

File: assets/js/cycles.js (hunk 1 above)
Comment (MichalD96) [reply assets/js/cycles.js]: ok, that is not needed

File: assets/js/cycles.js (hunk 2 above)
Comment (MichalD96) [reply assets/js/cycles.js]: i added return and braces because first i wanted to translate position

**Comment by MichalD96** [comment]
you can also delete hours and minutes now

**Pull request merged**